app/buck2_forkserver/src/run/mod.rs

```diff
@@ -370,6 +436,20 @@ async fn kill_process_impl(
     }
 }
 
+#[cfg(windows)]
+fn kill_job(job_handle: &WindowsJobHandle) -> anyhow::Result<()> {
+    unsafe {
+        let terminate_job_result = winapi::um::jobapi2::TerminateJobObject(job_handle.get_handle(), 1);
+
+        if terminate_job_result == winapi::shared::minwindef::FALSE {
+            Err(anyhow::Error::msg("Failed to terminate job to kill"))
```

Comment:
We would usually use `anyhow::anyhow!("Failed to terminate job to kill")`

Reply:
I replaced all my usages of `anyhow::Error::msg` with `anyhow::anyhow!`.